README.md

```diff
@@ -266,52 +264,57 @@ See how to use at https://github.com/pactflow/actions/tree/main/publish-pact-fil
 
 All of the following code will be used exclusively for Providers. This will run the Pacts against the real Provider and either verify or fail validation on the Pact Broker.
 
-##### Create Unit Test
+### Create Unit Test
 
-Create a single unit test function. This will test a single consumer of the service.
+Create a single unit test function. This will test all defined consumers of the service.
 
-##### Start API
+```php
+public function testPactVerifyConsumers(): void
+{
+    $config = new VerifierConfig();
+    $config
+        ->setProviderName('someProvider') // Providers name to fetch.
+        ->setProviderVersion('1.0.0') // Providers version.
+        ->setProviderTags('prod' ,'dev')
+        ->setProviderBranch('main')
+        ->setScheme('http')
+        ->setHost('localhost')
+        ->setPort(58000)
+        ->setBasePath('/')
+        ->setStateChangeUrl(new Uri('http://localhost:58000/change-state'))
+        ->setBuildUrl(new Uri('http://build.domain.com'))
+        ->setFilterConsumerNames('someConsumer', 'otherConsumer')
+        ->setFilterDescription('Send POST to create')
+        ->setFilterNoState(true)
+        ->setFilterState('state')
+        ->setPublishResults(true)
+        ->setDisableSslVerification(true)
+        ->setStateChangeAsBody(false)
+        ->setStateChangeTeardown(true)
+        ->setRequestTimeout(500);
```

Comment:
I think that for the front-page README, this is quite verbose and might overwhelm some people.

If a lot of the options have sane defaults, I would simply ignore them and focus on the important configuration options only. If they need these additional configuration options, they can search the more in-depth docs.

Reply:
So I think I will try to use configurations from my personal projects.

Reply:
I updated the example code, copy from `UPGRADE-10.0.md` and my personal project